### ISSUE TITLE
Error submitting or showing-json with -MERGING- directory left over in /var/db/pkg

### ISSUE THREAD

**Issue by palica** [haxmeister/funtoo-reporter#60]
```
Unable to open /var/db/pkg/sys-kernel/-MERGING-openvz-rhel6-stable-2.6.32.042.092.2/SLOT: No such file or directory
 at /usr/sbin/funtoo-report line 146.
```

These files/directories were left over from a merge that didn't finish. I was unaware that they were even there. Maybe we could just skip dirs with -MERGING- as they could be incomplete.

**Comment by tejr** [comment]
This should be resolved by commit 1e2ea08.

**Comment by palica** [comment]
yes, this got fixed. Thank you
Can be closed.
